### DISCUSSION TITLE
SSL Certification

### DISCUSSION THREAD

**Discussion by Ronald-Agorsah** [wg-easy/wg-easy#830] [Ideas]
Why is it that there is no SSL Certification possivble via my A Record?

**Comment by pheiduck** [comment]
We recommend using a proxy behind Wireguard Easy to use SSL. Some users also use it over http so we have not implemented SSL by default.

**Comment by vithurshanselvarajah** [comment]
If you go down the docker-compose route you can configure a nginx proxy, wireguard and pihole all in a single file. 

The benefit being that with Nginx you can then get certs from lets encrypt and disable public access direcly to wg-easy so all traffic must pass through proxy (excluding actual VPN traffic)
